packages/client/src/encryption/GroupKey.ts

```diff
@@ -68,22 +68,22 @@ export class GroupKey {
 
     /** @internal */
     encryptNextGroupKey(nextGroupKey: GroupKey): EncryptedGroupKey {
-        return new EncryptedGroupKey(nextGroupKey.id, EncryptionUtil.encryptWithAES(nextGroupKey.data, this.data))
+        return new EncryptedGroupKey(nextGroupKey.id, hexToBinary(EncryptionUtil.encryptWithAES(nextGroupKey.data, this.data)))
```

Comment:
The `encryptWithAES` method could return `Buffer` or `Uint8Array` instead of string? And similarly `decryptWithAES` could get the binary input instead of string?

Reply:
AES encryption changes are here. I could extract them to a new PR. https://github.com/streamr-dev/network/pull/1827/files#diff-12fc8d398bfd4ecaddfae2289da7958c5d3c67c51a9d8f61524c9a043ee6e4b4

Reply:
Could cherry-pick to this PR as needed here. Separate PR is also good but maybe that requires some redundant extra work.

Reply:
It is only used for group keys here though. Adding it here will require binary translation for regular messages so I don't see it as much of a win